fetchmail_notify_error_to_sender/views/fetchmail_view.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record model="ir.ui.view" id="view_email_server_form">
+        <field name="name">fetchmail.server.form</field>
+        <field name="model">fetchmail.server</field>
+        <field name="inherit_id" ref="mail.view_email_server_form" />
+        <field name="arch" type="xml">
+            <field name="active" position="after">
+                <field name="error_notice_template_id" />
```

Comment:
Can you move this field to somewhere else.
![image](https://github.com/OCA/server-tools/assets/10028499/b0561c7b-d8fb-4e55-bfcd-88af03e52d2f)
![image](https://github.com/OCA/server-tools/assets/10028499/9134db27-f51e-44c1-81d1-2b5009e73ecc)